src/app/shared/object-detail/my-dspace-result-detail-element/item-detail-preview/item-detail-preview.component.ts

```diff
@@ -35,9 +34,9 @@ export class ItemDetailPreviewComponent {
   @Input() object: SearchResult<any>;
 
   /**
-   * Represent item's status
+   * Represent context
    */
-  @Input() status: MyDspaceItemStatusType;
+  @Input() context: Context;
```

Comment:
I'd rename it to badgeContext so it is clearer what it's referring to

Reply:
I agree with @atarix83 .  I like calling this the `badgeContext` to make it clear what this is referencing. This is not the Context of the *Item*.  It's the Context for the *badge*.